roles/moving_objects_between_buckets/README.md

```diff
@@ -0,0 +1,86 @@
+moving_objects_between_buckets
+==================
+
+A role to move objects from one S3 Bucket to another.
+
+Requirements
+------------
+
+AWS User Account with the following permission:
+
+* s3:HeadBucket
+* s3:GetBucketOwnershipControls
+* s3:ListObjectsV2
+* s3:ListBucket
+
+Role Variables
```

Comment:
variable names are fine, but imo a bit confusing, maybe we can use something like `source_bucket`, `dest_bucket`?